docker/install.sh

```diff
@@ -9,10 +10,17 @@ if [ -e "jenkins_env.sh" ]; then
 	source "jenkins_env.sh"
 fi
 
+# Assumes docker
+docker pull mysql/mysql-server:5.7
+docker run -p 3306:3306 --name mysql -e MYSQL_ROOT_PASSWORD=rawls-test -e MYSQL_USER=rawls-test -e MYSQL_PASSWORD=rawls-test -e MYSQL_DATABASE=testdb -d mysql/mysql-server:5.7
+
 # catch sbt issues separately
 sbt update && echo "sbt updated successfully."
 
-sbt -J-Xms4g -J-Xmx4g test
+# Assumes docker-machine
+sbt -J-Xms4g -J-Xmx4g test -Ddockermysql.ip=`docker-machine ip default`
```

Comment:
Thanks! The BEST and SLICKEST way to do this is with this: https://mktmp.io and their Jenkins plugin. It costs money. But I'm asking for academic pricing. 

There are other ways to do it, too (so don't worry) but not quite as slick (or ie, they require me to do actual work).